capa/helpers.py

```diff
@@ -71,7 +71,11 @@ def assert_never(value) -> NoReturn:
 
 
 def get_format_from_report(sample: Path) -> str:
```

Comment:
this function should return the report format, not the report itself

Reply:
I didn't understand. The function returns report format right - FORMAT_CAPE or FORMAT_UNKNOWN

Reply:
@mr-tz, as you suggested, I've created a helper function `load_json_from_path` that can load both plain and compressed JSON reports, and there are still a few `json.loads` references, but they are used on strings, not for loading JSON reports (if I'm not wrong)